ulog/sentry/sentry.go

```diff
@@ -44,6 +44,18 @@ var _zapToRavenMap = map[zap.Level]raven.Severity{
 	zap.FatalLevel: raven.FATAL,
 }
 
+// Configuration provides sentry DSN and other optional parameters for logging
+type Configuration struct {
+	DSN      string
+	MinLevel *string                `yaml:"min_level,omitempty"`
```

Comment:
why all the omit empty tags? iirc those are only useful if we serialize to YAML which we'd never do -- only deser for config